krun.py

```diff
@@ -187,6 +187,22 @@ def main(parser):
 
     attach_log_file(config, args.resume)
 
+    mail_recipients = config.MAIL_TO
+    if type(mail_recipients) is not list:
```

Comment:
Well I guess it could be another collection, but since I have not tested it, and since the config file is ours to define, I think enforcing a list is OK.